src/uwtools/rocoto.py

```diff
@@ -124,36 +124,36 @@ def _add_compound_time_string(self, e: Element, config: Any, tag: str) -> Elemen
             e.text = str(config)
         return e
 
-    def _add_metatask(self, e: Element, config: dict, taskname: str) -> None:
+    def _add_metatask(self, e: Element, config: dict, name_attr: str) -> None:
```

Comment:
Here and below, harmonize the name of this argument across several functions dealing with metatasks, tasks, and `<sh>` dependencies.